source/main/index.js

```diff
@@ -20,7 +20,8 @@ mainErrorHandler();
 
 log.info(`========== Daedalus is starting at ${new Date()} ==========`);
 
-log.info(`!!! Daedalus is running on ${os.platform()} version ${os.release()}
+log.info(`!!! Daedalus is running on version ${environment.version}#${environment.buildNumber}
```

Comment:
@daniloprates can you please modify this line so that it reads:
```
!!! ${environment.getBuildLabel} is running on
```

Reply:
@daniloprates it should be a function call - I have updated my original comment which is now correct `environment.getBuildLabel()`